### PR TITLE
libvirt_numa: handle numa nodeset utils

### PR DESCRIPTION
**Pull request by dzhengfy**
Some formats of numa nodeset are provided in provider cases, but
those formats require the host numa nodes continouse. The truth is
host numa nodes have big differences among available numa hosts.
The host numa nodes might be not continuous, so certain format is
not support, like 'x-y'. In this case, we have to use 'x,y' instead.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by kylazhang** [comment]
@dzhengfy do you plan to also support the situation like '1,2,3' & '1-3' & '1-3,^2'  ?

File: virttest/utils_libvirt/libvirt_numa.py [hunk 1]
```diff
@@ -79,3 +81,57 @@ def create_hmat_xml(vmxml, params):
     vmxml.sync()
 
     return vmxml
+
+
+def get_continuous_numa_node_index(node_list):
```
Comment (chunfuwen) [inline virttest/utils_libvirt/libvirt_numa.py:86]: I assume you want to get the index of first adjacent numa node.
it sounds like it is a little bit clearer if method name changed to get_first_continuous_numa_node_index or get_first_adjacent_numa_node_index

**Comment by dzhengfy** [comment]
> @dzhengfy do you plan to also support the situation like '1,2,3' & '1-3' & '1-3,^2' ?

no plan yet as the cases' requirement is not to test those complex format so far.

File: virttest/utils_libvirt/libvirt_numa.py (hunk 1 above)
Comment (dzhengfy) [reply virttest/utils_libvirt/libvirt_numa.py]: Updated with rename and move the function to internal

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by Yingshun — APPROVED** [review]

**Pull request merged**